### ISSUE TITLE
Provide a default stylesheet

### ISSUE THREAD

**Issue by tlvince** [eHealthAfrica/angular-eha.date-of-birth-or-age#9]
Given this directive provides a default template, it might be nice that a default stylesheet comes along with it.

**Comment by tthew** [comment]
Agreed. I suggest we have a discussion about / formalise our SCSS/CSS organisation strategies first though or this could get very messy very fast.
